charts/airbyte/templates/env-configmap.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
```

Comment:
We should split this into a config map and secrets map (like https://github.com/airbytehq/airbyte/pull/5962)

Reply:
Also, should we be making all of these configurable from the values file @davinchia ?

Reply:
Yes we should make this configurable. 

I can handle splitting this out in a follow up PR.